ensime_shared/launcher.py

```diff
@@ -113,14 +127,21 @@ def generate_classpath(self, scala_version, classpath_file):
         project_dir = self.classpath_project_dir(scala_version)
         Util.mkdir_p(project_dir)
         Util.mkdir_p(os.path.join(project_dir, "project"))
-        Util.write_file(os.path.join(project_dir, "build.sbt"), self.build_sbt(scala_version, classpath_file))
-        Util.write_file(os.path.join(project_dir, "project", "build.properties"),
-                "sbt.version={}".format(self.sbt_version))
-        log_file = os.path.join(project_dir, "build.log")
-        log = open(log_file, 'w')
-        null = open("/dev/null", "r")
-        # see https://github.com/ensime/ensime-vim/issues/29 on why we use this method
-        self.vim.command("!(cd {};sbt -Dsbt.log.noformat=true -batch saveClasspath)".format(project_dir))
+        Util.write_file(
+            os.path.join(project_dir, "build.sbt"),
```

Comment:
Not 100% what your preferred solution is. (`\` is only needed in Python if there are no enclosing parentheses, brackets or braces, but again, I'm not entirely sure what you're proposing to change.)